maat-court-data-api/src/test/java/gov/uk/courtdata/assessment/controller/PassportAssessmentControllerTest.java

```diff
@@ -121,19 +120,13 @@ public void givenIncorrectParameters_whenUpdatePassportAssessmentIsInvoked_thenE
                 .andExpect(status().is4xxClientError());
     }
 
-    @Test
-    public void givenCorrectParameters_whenDeletePassportAssessmentIsInvoked_thenPassportAssessmentIsDeleted() throws Exception {
-        when(passportAssessmentValidationProcessor.validate(any(Integer.class))).thenReturn(Optional.empty());
-        doNothing().when(passportAssessmentService).delete(MOCK_ASSESSMENT_ID);
-        mvc.perform(MockMvcRequestBuilders.delete(endpointUrl + "/" + MOCK_ASSESSMENT_ID))
-                .andExpect(status().isOk());
-    }
 
     @Test
     public void givenIncorrectParameters_whenDeletePassportAssessmentIsInvoked_thenErrorIsThrown() throws Exception {
```

Comment:
This function should probably be renamed or removed. Does it make sense to have a test when there is no controller method?

Reply:
I was trying to assert that calls to delete will fail but I have removed this now